database/migrations/sql/20180604_08_create_visits.sql

```diff
@@ -5,6 +5,9 @@
 CREATE TABLE visit_activity_category (
   visit_activity_category_id   SERIAL NOT NULL UNIQUE,
   visit_activity_category_name VARCHAR(255) NOT NULL,
+  created_at                   TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  modified_at                  TIMESTAMP WITH TIME ZONE,
```

Comment:
All of these `modified_at` columns need to have triggers added so they auto-update.